client/web/src/enterprise/campaigns/list/backend.ts

```diff
@@ -151,3 +153,21 @@ export const queryCampaignsByNamespace = ({
             }
         })
     )
+
+export const areCampaignsLicensed = (): Observable<boolean> =>
+    requestGraphQL<AreCampaignsLicensedResult, AreCampaignsLicensedVariables>(
+        gql`
+            query AreCampaignsLicensed {
+                site {
+                    productSubscription {
+                        license {
+                            tags
+                        }
+                    }
+                }
+            }
+        `
+    ).pipe(
+        map(dataOrThrowErrors),
+        map(data => !!data.site?.productSubscription?.license?.tags.includes('campaigns'))
```

Comment:
According to the GraphQL schema, this should be sufficient
```suggestion
        map(data => !!data.site.productSubscription.license?.tags.includes('campaigns'))
```